src/topology/algebra/ordered.lean

```diff
@@ -308,69 +307,78 @@ induced_orderable_topology' f @hf
   (λ a x ax, let ⟨b, ab, bx⟩ := H ax in ⟨b, hf.1 ab, le_of_lt bx⟩)
 
 lemma nhds_top_orderable [topological_space α] [order_top α] [orderable_topology α] :
-  𝓝 (⊤:α) = (⨅l (h₂ : l < ⊤), principal {x | l < x}) :=
-by rw [@nhds_eq_orderable α _ _]; simp [(>)]
+  𝓝 (⊤:α) = (⨅l (h₂ : l < ⊤), principal (Ioi l)) :=
+by simp [nhds_eq_orderable (⊤:α)]
 
 lemma nhds_bot_orderable [topological_space α] [order_bot α] [orderable_topology α] :
-  𝓝 (⊥:α) = (⨅l (h₂ : ⊥ < l), principal {x | x < l}) :=
-by rw [@nhds_eq_orderable α _ _]; simp
+  𝓝 (⊥:α) = (⨅l (h₂ : ⊥ < l), principal (Iio l)) :=
+by simp [nhds_eq_orderable (⊥:α)]
 
 section linear_order
 
 variables [topological_space α] [linear_order α] [t : orderable_topology α]
 include t
 
-lemma mem_nhds_orderable_dest {a : α} {s : set α} (hs : s ∈ 𝓝 a) :
-  ((∃u, u>a) → ∃u, a < u ∧ ∀b, a ≤ b → b < u → b ∈ s) ∧
-  ((∃l, l<a) → ∃l, l < a ∧ ∀b, l < b → b ≤ a → b ∈ s) :=
+lemma mem_nhds_orderable_dest' {a : α} {s : set α} (hs : s ∈ 𝓝 a) :
```

Comment:
Since you are already changing this lemma, would you also consider splitting it in two lemmas instead of having a conjunction? Same thing for the next lemma.

Reply:
BTW, what `_dest` stands for in the name? Or how should I name the new lemmas?

Reply:
`dest` means `destruct`, i.e., start from an assumption and deduce a more handy formulation for it. It is standard Isabelle terminology, but not so much used in Lean. You could replace it by more standard Lean terminology, like `exist_Ico_subset_of_mem_nhds` or something like this.

Reply:
Done. I had to rewrite the proof (it seemed easier to me than adjusting the original one), and this resulted in adding more lemmas.

Reply:
Should I now split this PR into smaller pieces?

Reply:
No it's fine, I have already reviewed the rest, so I will accept this once I am happy with the name.